pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1104,6 +1104,33 @@ def test_time(self):
                     xp = time(h, m, s).strftime('%H:%M')
                 assert xp == rs
 
+    @pytest.mark.slow
+    @pytest.mark.xfail(strict=False, reason="Unreliable test")
+    def test_time_change_xlim(self):
+        t = datetime(1, 1, 1, 3, 30, 0)
+        deltas = np.random.randint(1, 20, 3).cumsum()
```

Comment:
I believe the random ints for `deltas` are what can cause the failures.

Reply:
Potentially, but probably best to investigate after this PR.